src/Logger/DefaultLogger.ts

```diff
@@ -25,6 +25,7 @@ export class DefaultLogger implements ILogger {
   }
 
   timestamp(): any {
-    return moment().format('HH:mm:ss');
+    return Math.floor(Date.now() / 1000);
```

Comment:
More reliable and cleaner would be to use `moment().unix()` if you need the unix timestamp.

Reply:
Ah yes, the reason I changed this was because it was causing an error when using the packaged index.js from the cli. Not sure why, will change it back and test again.